### PR TITLE
Document the new UX and UI for space member application

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Manage-Space.rst [hunk 1]
```diff
@@ -147,7 +147,7 @@ not its member yet, you will receive various instructions as follows:
 Space banner and avatar
 =======================
 
-After :ref:`creating a space <Create-Space>`, by default it has the default banner (in grey color) and the default space's icon. 
+After :ref:`creating a space <Create-Space>`, by default it has the default banner (related to the space template) and the default space's icon.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
After :ref:`creating a space <Create-Space>`, it has the default banner (related to the space template) space icon. 
```

File: docs/Manage-Space.rst [hunk 2]
```diff
@@ -156,78 +156,41 @@ Space banner
...
-
-|image16|
-
-Pick up an image by double-clicking on it. The image will be loaded in
-the space's avatar.
-
-After setting a space avtar, when you click again on |image71|, you can:
-
-- ``Update`` the image by selecting an another one from your computer or
+Contrary to the banner, you can choose an another space avatar only in your space **Settings**.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Unlike the banner, you can choose another space avatar only from space **Settings**.
```

File: docs/Manage-Space.rst [hunk 3]
```diff
@@ -156,78 +156,41 @@ Space banner
...
 -  The space's applications.
 
--  The *More* button |image18| appears when the number of applications
-   exceeds the number that the navigation bar is able to display or when
-   the spae's name is long.
-
-   |image19|
-
-   Clicking on it displays the list of the remaining applications.
+-  The same actions icons we already have in the platform default topbar:
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
-  The usual icons of the topbar: 
```

File: docs/Manage-Space.rst [hunk 4]
```diff
@@ -156,78 +156,41 @@ Space banner
...
-   |image19|
-
-   Clicking on it displays the list of the remaining applications.
+-  The same actions icons we already have in the platform default topbar: 
+   Notifications, Applications center, Search icon and Chat.
 
-   |image20|
-
-In mobile devices, applications navigation bar is scrollable right and
+In mobile devices, applications navigation bar is displayed on the bottom and scrollable right and
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
On mobile devices, application navigation bar is displayed at the bottom and can be scrolled from right to
```

File: docs/Manage-Space.rst [hunk 5]
```diff
@@ -578,117 +544,91 @@ Inviting new members
...
 
-If the invited users have not accepted your requests yet, you can revoke
-your invitations by clicking |image46| corresponding to the users' name.
-The users will be removed from the Invited list.
+Display Invited members
+-------------------------
 
-Validating/Declining request
------------------------------
+You can display the list of invited members just by selecting **Invited** on the filter space members pulldown.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
You can display the list of invited users just by selecting **Invited** on the filter space members pulldown.
```

users become members only after they are accepted. so invited members does not exist. it's invited users.

File: docs/Manage-Space.rst [hunk 6]
```diff
@@ -578,117 +544,91 @@ Inviting new members
...
 
-As a manager or creator of a space, you can validate other users'
-requests for joining your space.
+|SP-19|
 
--  To accept a user's request for joining your space, click |image47| in
-   the Action column.
+Display Requestd to join  members
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Display Requested to join  members
```

File: docs/Manage-Space.rst [hunk 7]
```diff
@@ -702,100 +642,57 @@ before you can delete that member.
...
-See :ref:`Editing a page <ManagingPages.EditingPage>` for more details.
-
-Removing a space page
-----------------------
-
-If you right-click the node containing the page and select Delete Node
-from the context menu, only the navigation node linking to the page will
-be removed from the space navigation bar, but its page still exists. To
-actually delete this page, see :ref:`Deleting a page <ManagingPages.DeletingPage>`.
+Every changement is automatically saved and the new applications order is available after a refresh.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Every change is automatically saved and the new applications order is available after a refresh.
```

**Review by plamarque — CHANGES_REQUESTED** [review]

**Comment by hghorbel** [comment]
@plamarque waiting for your review.

File: docs/Manage-Space.rst [hunk 8]
```diff
@@ -156,78 +156,41 @@ Space banner
...
-
-After setting a space avtar, when you click again on |image71|, you can:
-
-- ``Update`` the image by selecting an another one from your computer or
+Unlike the banner, you can choose another space avatar only from space **Settings**.
 
-- ``Reset`` to restore the default space icon.
 
-|image72|
+When scrolling down in the space's activity stream, or applications, you always have the space' avatar, the space name and their applications in the topbar.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
When scrolling down in the space's activity stream, or applications, you always have the space's avatar, the space name and their applications in the topbar.
```

File: docs/Manage-Space.rst [hunk 9]
```diff
@@ -340,7 +303,7 @@ Default Settings for the Community template are :
    -  Description: a general purpose area for collaboration and communication of a digital community.
    -  Hidden: No 
    -  Registration : Open
-   -  Predefined Apps : Home, Documents, Tasks, Forum, Wiki, Calendar, Members, Space Settings
+   -  Predefined Apps : Home, Documents, Tasks, Forum, Wiki, Calendar, Members, Settings
```
Comment (plamarque) [inline docs/Manage-Space.rst]: Wiki was renamed Notes

File: docs/Manage-Space.rst [hunk 10]
```diff
@@ -352,7 +315,7 @@ Projects is ideal to coordinate actors toward a shared outcome.
    -  Description: A project space where members coordinate on tasks toward a predefined outcome.
    -  Hidden: No 
    -  Registration : Validation
-   -  Predefined Apps : Home, Tasks, Documents, Calendar, Wiki, Forum, Members, Space Settings
+   -  Predefined Apps : Home, Tasks, Documents, Calendar, Wiki, Forum, Members, Settings
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   -  Predefined Apps : Home, Tasks, Documents, Calendar, Notes, Forum, Members, Settings
```

File: docs/Manage-Space.rst [hunk 11]
```diff
@@ -363,7 +326,7 @@ For organizational teams or work groups.
    -  Description: A central destination for members of a team.
    -  Hidden: Yes
    -  Registration : Validation
-   -  Predefined Apps : Home, Calendar, Documents, Wiki, Tasks, Members, Space Settings
+   -  Predefined Apps : Home, Calendar, Documents, Wiki, Tasks, Members, Settings
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   -  Predefined Apps : Home, Calendar, Documents, Notes, Tasks, Members, Settings
```

File: docs/Manage-Space.rst [hunk 12]
```diff
@@ -373,10 +336,10 @@ Communication is the perfect choice to exchange knowledges and skills.
    -  Description: Interpersonal communication where a space groups are involved in exchange of ideas, skills and interests.
    -  Hidden: No 
    -  Registration : Validation
-   -  Predefined Apps : Home, Documents, Tasks, Forum, Wiki, Calendar, Members, Space Settings
+   -  Predefined Apps : Home, Documents, Tasks, Forum, Wiki, Calendar, Members, Settings
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   -  Predefined Apps : Home, Documents, Tasks, Forum, Notes, Calendar, Members, Settings
```

File: docs/Manage-Space.rst [hunk 13]
```diff
@@ -402,15 +366,13 @@ The default space home page contains the following content:
...
+   |SP-11|
 
 .. note:: :ref:`Calendar <CalendarApp>` and :ref:`Who is Online? <WhoIsOnlineApp>` portlets are the same
           as for :ref:`intranet homepage <PLFHomepage>` but specific for the space's calendar and members.
 
 .. note:: When you are a  platform administrator and member of a space, you are able to add more portlets 
-          just by clicking on Edit --> Page --> Edit layout and then drag and drop the portelt you want to add.
-          
-              |SP-11|
+          just by clicking on Edit --> Page --> Edit layout and then drag and drop the portelt you want to add.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
          just by clicking on Edit --> Page --> Edit layout and then drag and drop the app you want to add.    
```

File: docs/Manage-Space.rst [hunk 14]
```diff
@@ -536,38 +505,35 @@ Changing space information/visibility
...
+This function allows you to edit the basic information of a space, which are General space informations and space's applications.
 
-1. Select the Settings tab in the **Space Settings** page.
+To open the Settings page, you can select the Settings tab in the space navigation bar.
 
-|image35|
+**General**
 
-2. Change information in the Name, Description fields and the space avatar.
+By clicking on the edit icon in front of the General label, uyou can change space's information like:
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
By clicking on the edit icon in front of the General label, you can change space's information like: 
```

File: docs/Manage-Space.rst [hunk 15]
```diff
@@ -578,117 +544,91 @@ Inviting new members
...
+|SP-20|
 
 .. _PromotingDemotingMember:
 
 Promoting/Demoting a member
 ----------------------------
 
--  To promote a member to the manager position, click |image49| in the
-   Manager column. The user will be automatically promoted as a manager
+-  To promote a member to the manager position, click **Promote Manager** in the
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
-  To grant a member the space's manager role, click **Promote Manager** in the
```

File: docs/Manage-Space.rst [hunk 16]
```diff
@@ -809,7 +706,7 @@ Joining/Leaving a space
 Joining a space
 ~~~~~~~~~~~~~~~~~~
 
-:ref:`Go to the Spaces page <note-access-spaces-page>` and you will see All Spaces tab which displays all your spaces and ones
+:ref:`Go to the Spaces page <Access-Space>` and you will see all your spaces and ones
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
:ref:`Go to the Spaces page <Access-Space>` and you will see all your spaces and those
```

**Review by plamarque — CHANGES_REQUESTED** [review]
typos and small languages fixes suggested

**Comment by hghorbel** [comment]
@plamarque  suggestions committed

File: docs/Manage-Space.rst (hunk 9 above)
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   -  Predefined Apps : Home, Documents, Tasks, Forum, Notes, Calendar, Members, Settings
```

**Review by plamarque — CHANGES_REQUESTED** [review]
one wiki occurrence was missed

**Comment by hghorbel** [comment]
Changes committed @plamarque

File: docs/Manage-Space.rst [hunk 17]
```diff
@@ -578,117 +544,91 @@ Inviting new members
...
------------------------------
+You can display the list of invited users just by selecting **Invited** on the filter space members pulldown.
 
-As a manager or creator of a space, you can validate other users'
-requests for joining your space.
+|SP-19|
 
--  To accept a user's request for joining your space, click |image47| in
-   the Action column.
+Display Requested to join  members
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Display Join Requests
```

File: docs/Manage-Space.rst [hunk 18]
```diff
@@ -702,100 +642,57 @@ before you can delete that member.
...
------------------------
-
-To create a page that is accessible on the space navigation bar, simply
-add a navigation node that links to your desired page in the Page
-Selector tab. See :ref:`Adding a new node <ManagingNavigations.AddingNewNode>` 
-for more details.
+|SP-26|
 
-|image57|
+Also, you can easily manage these applications display order on the space navigation bar
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Also, you can easily change the applications display order 
```

File: docs/Manage-Space.rst [hunk 19]
```diff
@@ -702,100 +642,57 @@ before you can delete that member.
...
+by selecting choices between: *Move After* or *Move Before*.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
with *Move After* or *Move Before*.
```

File: docs/Manage-Space.rst [hunk 20]
```diff
@@ -702,100 +642,57 @@ before you can delete that member.
...
-See :ref:`Editing a page <ManagingPages.EditingPage>` for more details.
-
-Removing a space page
-----------------------
-
-If you right-click the node containing the page and select Delete Node
-from the context menu, only the navigation node linking to the page will
-be removed from the space navigation bar, but its page still exists. To
-actually delete this page, see :ref:`Deleting a page <ManagingPages.DeletingPage>`.
+Every change is automatically saved and the new applications order is available after a refresh.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
All changes are automatically saved and the new order is applied after reloading the page.
```

File: docs/Manage-Space.rst [hunk 21]
```diff
@@ -844,10 +741,12 @@ you cannot leave a space will appear as below.
 
 |image61|
 
- .. note::After you have left a space, the space will not exist in the My
+ .. note: After you have left a space, the space will not exist in the My
 			Spaces tab, but in the All Spaces tab (for the "visible" space
 			only). You are not able to view activities of the spaces you have
 			left unless those where you have been mentioned.
+      Once you leave the space, you will not be able to see any activity, nor
+      receiving notifications of your former publications on this space
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
      receiving notifications of your former publications done in this space
```

File: docs/Manage-Space.rst [hunk 22]
```diff
@@ -919,15 +818,9 @@ with the space name, the space avatar and the space description.
 |image66|
 
 If you are a member in the space and not an administrator or creator, in
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
If you are a member in the space and not an administrator
```

File: docs/Manage-Space.rst [hunk 23]
```diff
@@ -919,15 +818,9 @@ with the space name, the space avatar and the space description.
...
-addition to the previously cited components, a Leave button is displayed
-in the popover allowing you to leave the space and Chat button allowing
+addition to the previously cited components, a Chat button allowing
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
you will see a Chat button allowing
```

**Review by plamarque — CHANGES_REQUESTED** [review]
more language enhancements missed in previous reviews.

**Comment by hghorbel** [comment]
@plamarque  Changes done.

**Comment by hghorbel** [comment]
ping @plamarque need your approve to merge it.

**Comment by hghorbel** [comment]
@plamarque ping

**Review by plamarque — APPROVED** [review]

**Pull request merged**